logstash.tf

```diff
@@ -1,32 +1,44 @@
 resource "aws_instance" "logstash" {
   ami                    = data.aws_ami.aws_ubuntu_image.id
   instance_type          = var.logstash_instance_type
-  subnet_id              = aws_subnet.private_subnet_name.id
+  subnet_id              = data.aws_subnet.subnet_g_ls_e.id
   vpc_security_group_ids = [aws_security_group.logstash_sg.id]
   key_name               = var.keypair_name
+  iam_instance_profile   = "instance-profile"
 
 
   tags = {
     Name = "logstash server"
   }
 
 }
-
-resource "aws_security_group" "logstash_sg" {
-    name="logstash_sg"
-    descritption= "SG for the Logtstash server"
-    vpc_id = data.aws_vpc.main.id
+resource "aws_volume_attachment" "logstash_ebs_attach" {
+  device_name = "/dev/sdh"
+  volume_id   = aws_ebs_volume.ebs_logstash.id
+  instance_id = aws_instance.logstash.id
 }
+resource "aws_security_group" "logstash_sg" {
+  name         = "logstash_sg"
+  description = "SG for the Logtstash server"
+  vpc_id       = data.aws_vpc.main.id
 
-#Inbound connection
-ingress{
-    description="beats port"
-    from_port= 5503
-    to_port=5503
-    protocol="tcp"
-    cidr_blocks=["0.0.0.0/0"]
-}
-    # 
+
+  #Inbound connection
+  ingress {
+    description = "beats port"
+    from_port   = 5044
```

Comment:
you asked for 5033 as it was aligned in the ticket and the diagram ...

Reply:
According with Ashley we've found out that the correct port is 5044, so on the diagram was ok. The ticket has been amended as well.

Reply:
ok, lucidchart altered again...